src/java_bytecode/java_bytecode_language.cpp

```diff
@@ -206,6 +207,24 @@ bool java_bytecode_languaget::typecheck(
   if(string_refinement_enabled)
     string_preprocess.initialize_conversion_table();
 
+  // Must load java.lang.Object first to avoid stubbing
+  // FIXME: its unclear if this ordering has to be done
+  // here or hard coded inside the classloader.
```

Comment:
File an issue to track this?

Reply:
@dcattaruzza Unless @tautschnig or @peterschrammel think it's better to move this code to the class loader, I propose to leave this code here and update the comment above to say that "This ordering could alternatively be enforced by moving the code below to the class loader."